js/theme/variables.js

```diff
@@ -48,12 +48,16 @@ const customVariables = Object.assign(materialVariables, {
 
   // Footer
   footerBackground: '#FFFFFF',
-  footerShadowWidth: 1,
-  footerShadowColor: '#D8D8D8',
+  footerElevation: 50,
   footerPaddingTop: 24,
   footerPaddingLeft: 24,
   footerPaddingBottom: 16,
   footerPaddingRight: 24,
+  footerShadowColor: '#000000',
```

Comment:
As a general pratice, I'd try to group related CSS properties in small reusable utilities in a similar way as implemented in frameworks like SuitCSS or [tachyons](http://tachyons.io/) or [styled-system](https://github.com/jxnblk/styled-system).

In the specific case I mean that it would be useful to have a generic shadow utility in some style/shadows.ts file:

```
export const shadow-1 = {
   shadowOffset: {
      width: 0,
      height: 50
   },
   shadowOpacity: 0.3,
   shadowRadius: 50
}
```
and use it like:

```
'.footer': {
 <other styles>,
 ...shadow-1
}
```

The same should state for every other property like paddings / margins (let's call it 'space', see how the space variable is used here: https://github.com/jxnblk/styled-system#getting-started), colors, font-size and so on. 

ie. paddings would became:

```
footerPaddingTop: padding-3,
footerPaddingBottom: padding-2,
```

and so on.

this helps keep styling **consistent** (spacing-units, shadows, font-size, ...) without spreading **magic numbers** in each component which would be otherwise hard to refactor.

@francescozaia @matteodesanti what do you think about this ?

Reply:
this is helpful but goes out of the scope of this PR, I would move the discussion to Pivotal, would you open a new story?

Reply:
https://www.pivotaltracker.com/story/show/156781049